src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -1262,11 +1262,17 @@ def collect_external_symbols(definition):
         return nonlocal_symbols, imported_symbols
 
     @staticmethod
-    def eval_constant(name: str, context: dict, loc=None):
+    def eval_external(name: str, context: dict, loc=None):
```

Comment:
Renamed `eval_constant` to `eval_external` to better indicate the scope of what it performs for the frontend.